### PR TITLE
Default text prop for Spinner component (fixes #154)

### LINKED ISSUE #154: Get rid of the string "Setting up"
Modify Spinner to make Loading the default.

### PR DESCRIPTION
**Pull request by muffinresearch**

### CODE REVIEW

File: public/jsx/views/card-details.jsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ module.exports = React.createClass({
         </div>
       );
     } else {
-      return <Spinner text={gettext('Setting up')}/>;
```
Comment (andymckay) [inline public/jsx/views/card-details.jsx]: Is there any difference to doing the gettext in the class as opposed to in the JSX? It looks like more code, but the result is same.

File: tests/test.spinner.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+describe('Spinner', function() {
+
+  it('Uses loading as the default text', function() {
+    var Spinner_ = TestUtils.renderIntoDocument(<Spinner />);
+    var textNode = helpers.findByClass(Spinner_, 'text');
+    assert.equal(textNode.getDOMNode().firstChild.nodeValue, 'Loading');
+  });
+
+  it('Uses custom text as supplied', function() {
+    var Spinner_ = TestUtils.renderIntoDocument(<Spinner text="whatever" />);
```
Comment (andymckay) [inline tests/test.spinner.jsx]: Ah and this test demonstrates the difference, you can pass a custom value.

**Comment by andymckay** [comment]
r+

File: public/jsx/views/card-details.jsx (hunk 1 above)
Comment (muffinresearch) [inline public/jsx/views/card-details.jsx]: The one in `getDefaultProps` is just a default which can be overridden by the component rendering Spinner as a prop.

**Pull request merged**